Source/Common/ArgumentLoader.cpp

```diff
@@ -215,6 +217,9 @@ namespace FEX::ArgLoader {
 
       if (Options.is_set_by_user("Threads")) {
         uint64_t Config = Options.get("Threads");
+        if (Config == 0) {
+          Config = get_nprocs();
```

Comment:
You probably want `get_nprocs_conf` due to how aggressively ARM devices park CPU cores.
`get_nprocs` will only return the current number of online cores.